elastic/apm-server
# Generate fields for APM package

As title. For more details, head to the diff in the `Readme.md` file.

## Motivation/summary

## Checklist


I have considered changes for:

## How to test these changes

- run `make update`
- check the contents of `apmpackage/apm`

## Related issues

#4004
